Final_Project/Main.cpp

```diff
@@ -179,6 +179,10 @@ LRESULT CALLBACK WndProc(HWND hwnd, UINT Message,
 			while ((temp = getPayload(&currUploadFile))[0] != EOF) {
 				MessageBox(hwnd, temp, "title", MB_OK);
 			}
+
+			char ctrlFrame[4] = {};
+			sendFrame(ctrlFrame, NULL, ENQ);
+			ENQ_FLAG = true;
 			
```

Comment:
I tested by adding receiveFrame(ctrlFrame), and it worked beautifully. 
integrated well with alex's concede line.
Good job!